{{cookiecutter.project_slug}}/docs/conf.py

```diff
@@ -14,11 +14,20 @@
 import sys
 import django
 
-{% if cookiecutter.use_docker == 'y' %}
-sys.path.insert(0, os.path.abspath("/app"))
-os.environ.setdefault("DATABASE_URL", "")
-{% else %}
-sys.path.insert(0, os.path.abspath(".."))
+if os.getenv("READTHEDOCS", default=False) == "True":
+    sys.path.insert(0, os.path.abspath(".."))
+    os.environ["DJANGO_READ_DOT_ENV_FILE"] = "True"
+    {%- if cookiecutter.use_celery == 'y' %}
+    os.environ["CELERY_BROKER_URL"] = os.getenv("REDIS_URL", "redis://redis:6379")
+    {%- endif %}
+    os.environ["USE_DOCKER"] = "no"
+    master_doc = "_source/index"
```

Comment:
So if I follow, it's partly because of this line that we have this problem:

> you must go to /_source for this to work due to how the docs/* configuration is setup up.

But if we remove it the build fails because the source isn't in the right place relative to `conf.py`.

To fix it, we should move the `conf.py` in `_source`. We'll need to update a few other things to make it work, but I managed to make the RTD part work:

https://github.com/browniebroke/django-rtd
https://readthedocs.org/projects/django-rtd-tests/builds/

Reply:
That's awesome! I'll take a look and try to add your conf.py to this.